web/gui/dashboard_info.js

```diff
@@ -3012,4 +3017,43 @@ netdataDashboard.context = {
         info: 'Size of metric samples written to disk.'
     },
 
+    // ------------------------------------------------------------------------
+    // eBPF
+
+    'ebpf.file_descriptor': {
```

Comment:
Should it bee `ebpf` or `ebpf_something`? I think you are going to have several ebpf plugins, right?

Reply:
Yes, we will have different plugins. My initial thought was to have everything inside the the `eBPF` menu, but they would use different groups. 
We can talk more about this tomorrow during our meeting.
Thank you for the suggestion!